tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -284,13 +291,13 @@ def set_throughput_streams():
             if batch_size == 0:
                 batch_size = 1
 
-        # --------------------- 8. Setting optimal runtime parameters --------------------------------------------------
+        # --------------------- 8. Querying optimal runtime parameters --------------------------------------------------
         next_step()
-
-        # Update number of streams
-        for device in device_number_streams.keys():
-            key = device + '_THROUGHPUT_STREAMS'
-            device_number_streams[device] = benchmark.ie.get_config(device, key)
+        if not is_flag_set_in_command_line('hint'):
```

Comment:
I think we need report real number of streams for hint mode as well

Reply:
yes, I fixed the code to get the #streams from the exec_network instead and it works now